dfget/core/api/download_api.go

```diff
@@ -54,6 +55,7 @@ func (d *downloadAPI) Download(ip string, port int, req *DownloadRequest) (*http
 	headers[config.StrRange] = config.StrBytes + "=" + req.PieceRange
 	headers[config.StrPieceNum] = strconv.Itoa(req.PieceNum)
 	headers[config.StrPieceSize] = fmt.Sprint(req.PieceSize)
+	headers[config.StrUserAgent] = "dfget/" + version.DFGetVersion
```

Comment:
Regardless of `Supernode`, uploader does not parse this header  currently.

Reply:
`User-Agent` is a standard http header, it's convenient for us to know where the request comes from.